cli/server_report_command.go

```diff
@@ -956,52 +956,54 @@ func (c *SrvReportCmd) reportJetStream(_ *fisk.ParseContext) error {
 	switch {
 	case c.isFiltered():
 	case expectedClusterSize == 0:
-	case len(jszResponses) > 0 && cluster == nil:
+	case len(jszResponses) > 0 && len(clusters) == 0:
 		fmt.Println()
 		fmt.Printf("WARNING: No cluster meta leader found. The cluster expects %d nodes but only %d responded. JetStream operation requires at least %d up nodes.", expectedClusterSize, len(jszResponses), expectedClusterSize/2+1)
 		fmt.Println()
 	default:
-		cluster.Replicas = append(cluster.Replicas, &server.PeerInfo{
-			Name:    cluster.Leader,
-			Current: true,
-			Offline: false,
-			Active:  0,
-			Lag:     0,
-		})
-
-		sort.Slice(cluster.Replicas, func(i, j int) bool {
-			return cluster.Replicas[i].Name < cluster.Replicas[j].Name
-		})
+		for _, cluster := range clusters {
+			cluster.Replicas = append(cluster.Replicas, &server.PeerInfo{
+				Name:    cluster.Leader,
+				Current: true,
+				Offline: false,
+				Active:  0,
+				Lag:     0,
+			})
 
-		names := []string{}
-		for _, r := range cluster.Replicas {
-			names = append(names, r.Name)
-		}
-		if c.compact {
-			cNames = iu.CompactStrings(names)
-		} else {
-			cNames = names
-		}
+			sort.Slice(cluster.Replicas, func(i, j int) bool {
+				return cluster.Replicas[i].Name < cluster.Replicas[j].Name
+			})
 
-		table := iu.NewTableWriterf(opts(), "RAFT Meta Group Information")
-		table.AddHeaders("Connection Name", "ID", "Leader", "Current", "Online", "Active", "Lag")
-		for i, replica := range cluster.Replicas {
-			leader := ""
-			peer := replica.Peer
-			if replica.Name == cluster.Leader {
-				leader = "yes"
-				peer = cluster.Peer
+			names := []string{}
+			for _, r := range cluster.Replicas {
+				names = append(names, r.Name)
 			}
+			if c.compact {
+				cNames = iu.CompactStrings(names)
+			} else {
+				cNames = names
+			}
+
+			table := iu.NewTableWriterf(opts(), "RAFT Meta Group Information")
+			table.AddHeaders("Connection Name", "ID", "Leader", "Current", "Online", "Active", "Lag")
+			for i, replica := range cluster.Replicas {
+				leader := ""
+				peer := replica.Peer
+				if replica.Name == cluster.Leader {
+					leader = "yes"
+					peer = cluster.Peer
+				}
 
-			online := "true"
-			if replica.Offline {
-				online = color.New(color.Bold).Sprint("false")
+				online := "true"
+				if replica.Offline {
+					online = color.New(color.Bold).Sprint("false")
+				}
+
+				table.AddRow(cNames[i], peer, leader, replica.Current, online, f(replica.Active), f(replica.Lag))
 			}
 
-			table.AddRow(cNames[i], peer, leader, replica.Current, online, f(replica.Active), f(replica.Lag))
+			fmt.Print(table.Render())
```

Comment:
Might want to add a check here and print a newline if we are going to print multiple tables, just to give it some space on the screen.